### PR TITLE
limesuite 23.10.0 (new formula)

### PR DESCRIPTION
**Pull request by tesch1**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/l/limesuite.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      -DENABLE_API_DOXYGEN=OFF
+      -DDOWNLOAD_IMAGES=TRUE
+      -DLIME_SUITE_EXTVER=release
+      -DLIME_SUITE_ROOT='#{HOMEBREW_PREFIX}'
+      -DSOAPY_SDR_ROOT=#{HOMEBREW_PREFIX}
+      -DCMAKE_INSTALL_RPATH=#{rpath}
+      -DCMAKE_INSTALL_NAME_DIR=#{lib}
+    ]
+
+    args += ["-DCMAKE_INSTALL_NAME_DIR=#{lib}"]
```
Comment (chenrui333) [inline Formula/l/limesuite.rb:35]: this line is dup of L32?

File: Formula/l/limesuite.rb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args, *args
```
Comment (chenrui333) [inline Formula/l/limesuite.rb:36]:
```suggestion
    system "cmake", "-S", ".", "-B", "build", *args, *std_cmake_args
```

**Review by chenrui333 — COMMENTED** [review]
lgtm with some minor comments

File: Formula/l/limesuite.rb (hunk 2 above)
Comment (tesch1) [reply Formula/l/limesuite.rb]: Change made.

File: Formula/l/limesuite.rb (hunk 1 above)
Comment (tesch1) [reply Formula/l/limesuite.rb]: yep good call.  Fixed.

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
@tesch1, thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request merged**